src/StockportWebapp/wwwroot/assets/sass/stockportgov/modules/_featureCard.scss

```diff
@@ -118,8 +117,9 @@
             height: 300px;
         }
 
-        @media #{$tablet} {
-            max-height: 450px;
+        @media #{$tablet, $mobile} {
```

Comment:
No need to specify both tablet and mobile breakpoints. Because of the way the breakpoints work, specifying tablet will apply the styles for both mobile and tablet devices